packages/@aws-cdk/aws-ec2/README.md

```diff
@@ -252,6 +252,21 @@ DatabaseSubnet1   |`ISOLATED`|`10.0.6.0/28` |#1|Only routes within the VPC
 DatabaseSubnet2   |`ISOLATED`|`10.0.6.16/28`|#2|Only routes within the VPC
 DatabaseSubnet3   |`ISOLATED`|`10.0.6.32/28`|#3|Only routes within the VPC
 
+### Accessing the Internet Gateway
+
+If you need access to the internet gateway, you can get it's ID like so:
+
+```ts
+const igw_id = vpc.internetGatewayId;
+```
+
+This is only supported for VPC's created in the stack - currently you're 
+unable to get the ID for imported VPC's. To do that you'd have to use the
```

Comment:
How would you use an escape hatch to access the id for an imported VPC? I don't believe you can, as this information would need to be actively looked up...

Reply:
Changed wording in c38d4fb.

Reply:
👍